src/business-logic/services/test/rundown-timeline-service.spec.ts

```diff
@@ -55,6 +56,22 @@ describe(RundownTimelineService.name, () => {
 
       await expect(() => testee.deleteRundown(rundown.id)).rejects.toThrow(ActiveRundownException)
     })
+
+    it('throws an exception, when trying to active a rundown when there is another already activated rundown', async () => {
```

Comment:
You have put this test under the `describe` for the `deleteRundown` method.
You should make a new `describe` for the `activateRundown` method to put this test in.
Something like this:
```
describe(RundownTimelineService.name, () => {
  describe(RundownTimelineService.prototype.activateRundown.name, () => {
    // Your test here
  })

  describe(RundownTimelineService.prototype.deleteRundown.name, () => {
    ...
    // This is where you currently have your test.
  })
})
```

Reply:
correct.